git-publish

```diff
@@ -632,6 +634,12 @@ def main():
         cc_cmd = git_get_config('branch', topic, 'gitpublishcccmd') or \
                  get_profile_var(options.profile_name, 'cccmd')
 
+    blurb_template = options.blurb_template
+    if not blurb_template:
+        blurb_template = '\n'.join(get_profile_var_list(options.profile_name, 'blurb-template'))
```

Comment:
So, I'm using get_profile_var_list instead of get_profile_var because it allows multiple lines.
Things like \n or defining more than one blurb-template field would work fine.